### DISCUSSION TITLE
新建的test.js 无法调试

### DISCUSSION THREAD

**Discussion by codesopen** [whyour/qinglong#1655] [Q&A]
青龙版本v2.13.3
![image](https://user-images.githubusercontent.com/64585660/196320213-9ad7567b-3bb7-42f0-878c-d9134b303557.png)

**Comment by whyour** [comment]
> 青龙版本v2.13.3
> 
> ![image](https://user-images.githubusercontent.com/64585660/196320213-9ad7567b-3bb7-42f0-878c-d9134b303557.png)
> 
> 

最新版修了

**Reply by codesopen** [reply]
问题依然存在，麻烦大佬帮我看看是哪里的问题

**Comment by codesopen** [comment]
青龙版本v2.14.7还是有这个问题
![image](https://user-images.githubusercontent.com/64585660/196856359-976037b6-940c-4ca2-b6cc-526e7ea6da33.png)

**Reply by whyour** [reply]
> 青龙版本v2.14.7还是有这个问题 ![image](https://user-images.githubusercontent.com/64585660/196856359-976037b6-940c-4ca2-b6cc-526e7ea6da33.png)

看你是不是websocket连接失败了

**Reply by codesopen** [reply]
是这个问题。 后来我发现是nginx代理导致socke失败。 非常感谢！
